Formula/jj.rb

```diff
@@ -0,0 +1,24 @@
+class Jj < Formula
+  desc "Jujutsu is a Git-compatible DVCS"
```

Comment:
```suggestion
  desc "Git-compatible DVCS"
```
Can we also spell out `DVCS` instead of using an acronym?